toolchain/check/testdata/operators/overloaded/make_tests.sh

```diff
@@ -0,0 +1,38 @@
+#! /bin/bash
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+make_test() {
+  HEADER="// This file was generated from $4. Run make_tests.sh to regenerate."
+  sed "s,INTERFACE,$1,g; s,OP,$2,g; s,HEADER,$HEADER," $4 > $3.carbon
+}
```

Comment:
Just to note, discussed this offline: agreed to a TODO to remove the script, with the expectation that the prelude would lead to these being pretty small and not worth autoupdate anymore.

Reply:
Per discussion, added a TODO to say the script is short-term and should be removed once things stabilize here.